CryptoLightNews/MainTabBarController.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  MainTabBarController.swift
+//  CryptoLightNews
+//
+//  Created by Richard Price on 15/12/2022.
+//
+
+import UIKit
+
+class MainTabBarController: UITabBarController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupUI()
+        view.backgroundColor = .darkGray
+    }
+    
+    private func setupNavControllers(with rootViewController: UIViewController, title: String, image: UIImage) -> UINavigationController {
+        let navController = UINavigationController(rootViewController: rootViewController)
+        rootViewController.title = title
+        navController.tabBarItem.title = title
+        navController.tabBarItem.image = image
+        return navController
+    }
+    
+    private func setupUI() {
+        viewControllers = [
+            setupNavControllers(with: UIViewController(), title: "News", image: #imageLiteral(resourceName: "cryptoNews")),
+            setupNavControllers(with: UIViewController(), title: "Prices", image: #imageLiteral(resourceName: "prices_new")),
+        ]
+        //modify tab bar insets
+                guard let items = tabBar.items else {return}
+                for item in items {
+                    item.title = ""
+                    item.imageInsets = UIEdgeInsets(top: 2, left: 0, bottom: -2, right: 0)
```

Comment:
When we have constants like this, we tend to have them in a single place that we can use, and it's easier to change. We generally use enumerations to hold these constants. And we declare these enumerations at the beginning of the class. Have a look at this [article](https://www.swiftbysundell.com/articles/powerful-ways-to-use-swift-enums/) to get more context.
In this case, we can have two different approaches(please, feel free to choose one of them):
1 - Create constants of the numbers:
```
  enum Sizes {
    static let tapSpace = 2.0
  }
```
And then you use it like this:
```suggestion
 item.imageInsets = UIEdgeInsets(top: Sizes.tapSpace, left: .zero, bottom: -Sizes.tapSpace, right: .zero)
```
(.zero comes with Swift)

2 - The other way is to set a constant of the inset, like this:
```
  enum Sizes {
    static let tabInsets = UIEdgeInsets(top: 2, left: .zero, bottom: -2, right: .zero)
  }
```
And then use it like this: 
```suggestion
  item.imageInsets = Sizes.tabInsets
```